test/python/rest/string_tests.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
I'm confused, what is this test to do with this project?

Reply:
This comes from the example in the README file. It didn't work, so I copied the example into this source file, fixed it, tested it, and copied it back into the README file. I could delete the source file, but think working examples are more useful than commentary examples, so I'd rather replace the example in the README file with a link to this source file.